src/test/java/nextstep/subway/station/StationAcceptanceTest.java

```diff
@@ -31,85 +59,92 @@ public void setUp() {
     }
 
     /**
-     * When 지하철역을 생성하면
-     * Then 지하철역이 생성된다
-     * Then 지하철역 목록 조회 시 생성한 역을 찾을 수 있다
+     * When 지하철역을 생성하면 Then 지하철역이 생성된다 Then 지하철역 목록 조회 시 생성한 역을 찾을 수 있다
      */
     @DisplayName("지하철역을 생성한다.")
     @Test
     void createStation() {
         // when
-        Map<String, String> params = new HashMap<>();
-        params.put("name", "강남역");
-
-        ExtractableResponse<Response> response =
-                RestAssured.given().log().all()
-                        .body(params)
-                        .contentType(MediaType.APPLICATION_JSON_VALUE)
-                        .when().post("/stations")
-                        .then().log().all()
-                        .extract();
+        ExtractableResponse<Response> response = 지하철역_생성_요청("강남역");
 
         // then
-        assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value());
+        지하철역이_생성됨(response);
 
         // then
-        List<String> stationNames =
-                RestAssured.given().log().all()
-                        .when().get("/stations")
-                        .then().log().all()
-                        .extract().jsonPath().getList("name", String.class);
-        assertThat(stationNames).containsAnyOf("강남역");
+        List<String> 등록된_모든_지하철역_이름_목록 = 등록된_모든_지하철역_이름_조회();
+        목록에_주어진_역이_포함됨(등록된_모든_지하철역_이름_목록, "강남역");
+    }
+
+    private void 지하철역이_생성됨(ExtractableResponse<Response> response) {
+        assertAll(
+                () -> assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value()),
+                () -> assertThat(response.header("Location")).matches("/stations/" + 숫자_REGEX)
+        );
+    }
+
+    private void 목록에_주어진_역이_포함됨(List<String> 등록된_모든_지하철역_이름_목록, String 지하철역_이름) {
+        assertThat(등록된_모든_지하철역_이름_목록).containsAnyOf(지하철역_이름);
```

Comment:
```suggestion
        assertThat(등록된_모든_지하철역_이름_목록).contains(지하철역_이름);
```
`contains`가 더 적합하지 않을까요? 🤔

Reply:
하나니까 말씀해주신 게 맞다고 생각합니다! 수정할게요!